File: branches/Medusa/Source/Fantasy.Jobs/Tasks/XPathSetElementTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Linq;
using System.Xml;

namespace Fantasy.Jobs.Tasks
{
    [Task("xpathSetElement", Consts.XNamespaceURI, Description="Using XPath to set xml element value to xml file" )] 
    public class XPathSetElementTask : XPathSetValueTaskBase 
    {
        protected override void SetValue(XElement element, XmlNamespaceManager nsMgr)
        {
            element.Value = this.Value;
        }
    }
}

File: branches/Stheno/AutoConverted/Fantasy.Jobs/Tasks/XPathSetElementTask.java
package Fantasy.Jobs.Tasks;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[Task("xpathSetElement", Consts.XNamespaceURI, Description="Using XPath to set xml element value to xml file")]
public class XPathSetElementTask extends XPathSetValueTaskBase
{
	@Override
	protected void SetValue(XElement element, XmlNamespaceManager nsMgr)
	{
		element.setValue(this.getValue());
	}
}